console/console-init/ui/src/modules/project/components/ProjectList/ProjectList.tsx

```diff
@@ -67,27 +73,58 @@ export const ProjectList: React.FunctionComponent<IProjectListProps> = ({
   onEdit,
   onDelete,
   onDownload,
+  onEnable,
+  onDisable,
   onSelectProject
 }) => {
   const actionResolver = (rowData: IRowData) => {
     const originalData = rowData.originalData as IProject;
-    return [
-      {
-        id: "edit-project",
-        title: "Edit",
-        onClick: () => onEdit(originalData)
-      },
-      {
-        id: "delete-project",
-        title: "Delete",
-        onClick: () => onDelete(originalData)
-      },
-      {
-        id: "download-certificate-project",
-        title: "Download Certificate",
-        onClick: () => onDownload(originalData)
+    let actions = [];
+    if (originalData.projectType === ProjectTypes.IOT) {
+      actions = [
+        {
+          id: "delete-project",
+          title: "Delete",
+          onClick: () => onDelete(originalData)
+        }
+      ];
+      if (originalData.isEnabled !== undefined) {
```

Comment:
can you please try to reduce number of if else as much as possible.
you can replace if else here with turnary operator.
const {isEnabled}=originalData;
if (isEnabled !== undefined) {
         actions.push({
            id: isEnabled ? "disable-project": "enable-project",
            title: isEnabled ? "Disable" : "Enable",
            onClick: () => isEnabled ? onDisable(originalData) : onEnable(originalData))
          });
}